Vaca.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EjercicioDos
+{
+    internal class Vaca : Mamifero, Muerte
+    {    //se sobrescribe el metodo de mamimero amamantar
+        public override void amamanta()
+        {
+            Console.WriteLine("la vaca da leche");
+        }
+        //logica para calcular murte y pares
+        public string Morir(int nacimiento, int faleccimiento, string causa)
+        {
+            int calculo = nacimiento - faleccimiento;
+            calculo = calculo * 2;
+            string N;
```

Comment:
What is N? use a meaningful name

Reply:
a change has been made to the variable to be significant